tests/index.test.ts

```diff
@@ -440,21 +426,17 @@ describe("Test if the request is proxied to the designated service", () => {
       getMockOpenAI()
         .intercept({
           method: "POST",
-          path: "/v1/chat/completions",
+          path: `/v1/chat/completions?temperature=${0.1 * i}`,
           headers: {
             "content-type": "application/json",
             authorization: `Bearer ${BINDINGS["API_OPENAI_COM_API_KEY"]}`,
           },
-          body: JSON.stringify({
-            model: "gpt-3.5-turbo",
-            messages: [{ role: "user", content: `This is a test for ${i}!` }],
-            temperature: 0.1 * i,
-          }),
+          body: undefined,
```

Comment:
Confused why this is undefined now

Reply:
The `clone.body` forwarded to the proxied service is a readable stream and not a JSON-encoded string. Therefore, it was essential to set it as `undefined`, indicating that it can be of any type, such as a stream, string, or form data.

Reply:
Confused why we can't test the body anymore for temperature